### PR TITLE
Fixes #29888 - Add ';vdsmdummy;' and 'br-int' to interface exclusion lists

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

**Review by lzap — APPROVED** [review]
Yeah these only makes things slower.

File: app/models/setting/provisioning.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def self.local_boot_labels
     'usb*',
     'vnet*',
     'macvtap*',
-    '_vdsmdummy_',
```
Comment (ekohl) [inline app/models/setting/provisioning.rb:23]: Was the old one unused? From the subject I expected it to only add, not replace one.

Comment (adamruzicka) [reply app/models/setting/provisioning.rb:23]:
```
Additional info:
Current interface output of a RHV-Linux hypervisor with OVS enabled:

# ip l |egrep "vdsmdummy|br-int"
7: br-int: <BROADCAST,MULTICAST> mtu 1442 qdisc noop state DOWN mode DEFAULT group default qlen 1000
40: ;vdsmdummy;: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN mode DEFAULT group default qlen 1000
```

Looks like the one with underscores was wrong and the other was missing

Comment (lzap) [reply app/models/setting/provisioning.rb:23]: I also think so: https://github.com/oVirt/ovirt-engine/search?q=vdsmdummy&type=Code

**Comment by lzap** [comment]
Just to doublecheck: @ShimShtein aren't those underscores actually result of our parser/regexp implemntation of conversing special characters to underscore? Do you remember?

**Comment by ShimShtein** [comment]
@lzap, not that I am aware of. We only have `*` as a special character.

File: app/models/setting/provisioning.rb (hunk 1 above)
Comment (ekohl) [reply app/models/setting/provisioning.rb:23]: Looks like you're right. Looking at https://github.com/oVirt/vdsm/commit/a8708b8a8d7a1e7c5abde2b6b1cb8ad6c4555ddd#diff-4fda6a624b7ea6ccc1842e181c60b171R46 (dated in 2012) I think it's safe to assume the underscores version never existed.

**Comment by ekohl** [comment]
The only thing I can think of is that Facter normalized semi colons to underscores. Especially Facter 2 which didn't have structured facts. However, I can't reproduce that now:
```console
# facter --version
3.14.7
# ip link add ';vdsmdummy;' type bridge
# facter --show-legacy | grep vdsmdummy
interfaces => ;vdsmdummy;,enp0s31f6,lo,tap0,virbr0,virbr0-nic,wlp58s0
macaddress_;vdsmdummy; => 06:d1:7c:46:c9:de
mtu_;vdsmdummy; => 1500
    ;vdsmdummy; => {
```

**Comment by lzap** [comment]
Thanks for testing this, appreciated. That could be the reason indeed.

**Pull request merged**